README.md

```diff
@@ -58,7 +58,14 @@ npm run test
 tree-sitter parse test/test.just
 ```
 
-You can use the `nvim-treesitter/playground` plugin [from here](https://github.com/nvim-treesitter/playground), to explore the resulting parse tree. Use `TSPlaygroundToggle` to view the parse tree, and use `TSHighlightCapturesUnderCursor` to view highlight groups
+ALl our queries are in `queries-src`. This directory is what gets tested by
```

Comment:
A typo here. Also, treesitter playground plugin is deprecated, wince this functionality already baked into NeoVim (Inspect\InspectTree commands).

Reply:
Thanks, updated this in #55